addons/web/static/tests/views/pivot_tests.js

```diff
@@ -825,6 +825,80 @@ QUnit.module('Views', {
         pivot.destroy();
     });
 
+    QUnit.test('correctly remove pivot_ keys from the context', function (assert) {
+        assert.expect(5);
+
+        this.data.partner.fields.amount = {string: "Amount", type: "float"};
+
+        // Equivalent to loading with default filter
+        var pivot = createView({
+            View: PivotView,
+            model: "partner",
+            data: this.data,
+            arch: '<pivot>' +
+                        '<field name="date" interval="day" type="col"/>' +
+                        '<field name="amount" type="measure"/>' +
+                '</pivot>',
+            viewOptions: {
+                context: {
+                    pivot_measures: ['foo'],
+                    pivot_column_groupby: ['customer'],
+                    pivot_row_groupby: ['product_id'],
+                },
+            },
+        });
+
+        // Equivalent to unload the filter
+        var reloadParams = {
+            context: {},
+        };
+        pivot.reload(reloadParams);
+
+        assert.deepEqual(pivot.getContext(), {
+            pivot_column_groupby: ['customer'],
+            pivot_measures: ['foo'],
+            pivot_row_groupby: ['product_id'],
+        }, "context should be correct");
```

Comment:
is this what we want? If we removed the favorite, should those params be kept?

Reply:
@aab-odoo 
It makes sense to keep the groupbys, yes
You could have set up a filter with a domain, apply it. Remove the filter just to see the impact of the filter on the entire business.